tests/plugins/test_resolve_composes.py

```diff
@@ -360,13 +356,9 @@ def test_inherit_parents(self, mocked_env, parent_compose, parent_repourls,
 
         compose_ids = []
         current_repourl = ["http://example.com/current.repo"]
-        expected_yum_repourls = []
-        if repo_provided:
-            expected_yum_repourls = list(current_repourl)
+        expected_yum_repourls = defaultdict(set)
         if not ids and compose_defined:
-            expected_yum_repourls.append(ODCS_COMPOSE['result_repofile'])
-        if allow_inherit and parent_repourls:
-            expected_yum_repourls.append(parent_repo)
+            expected_yum_repourls[ODCS_COMPOSE_DEFAULT_ARCH].add(ODCS_COMPOSE['result_repofile'])
```

Comment:
since you are adding always only default arch x86

change for consistency also : arches = ['ppc64le', 'x86_64']

because that ppc is basically ignored, because you request composes for both arches and return compose just for x86

Reply:
or just improve that compose_defined scenario and return results for both arches

Reply:
updated the test to include both arches.